src/notification/toast-notification.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, Input, OnInit } from "@angular/core";
+
+import { NotificationService } from "./notification.service";
+import { InlineNotificationContent } from "./notification-content.interface";
+import { InlineNotification } from "./inline-notification.component";
+
+/**
+ * Notification messages are displayed toward the top of the UI and do not interrupt user’s work.
+ *
+ * @export
+ * @class Notification
+ */
+@Component({
+	selector: "ibm-toast-notification",
```

Comment:
This should just be `ibm-toast` ... We can just export it from the banner module since the external API doesn't change.